Cargo.toml

```diff
@@ -1,44 +1,53 @@
 [package]
-name = "tuf"
-version = "0.0.2"
-authors = [ "heartsucker <heartsucker@autistici.org>" ]
+authors = ["heartsucker <heartsucker@autistici.org>"]
```

Comment:
I'm not sure what you did, but can you leave these fields in the same order as there were originally. I don't see any reason to change their order.